python-package-manager/get_package_version/get_version.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+version_regex="[0-9a-zA-Z\.]+"
+
+if [[ -n $INPUT_PACKAGE_MANAGER ]]
+then
+    if [[ "$INPUT_PACKAGE_MANAGER" == "pipenv" ]]
+    then
+        cat Pipfile.lock | jq --raw-output '."'"$INPUT_GROUP"'"."'"$INPUT_PACKAGE"'".version' | grep -Po $version_regex
```

Comment:
Is `jq` installed by default?

Reply:
It is yeh, I was half expecting it to not be.

Reply:
Nice